src/components/Tipseen/TipseenBasicContent.tsx

```diff
@@ -3,23 +3,38 @@ import cx from "classnames";
 import TipseenTitle from "./TipseenTitle";
 import { VibeComponentProps } from "../../types";
 import { ElementContent } from "../../types/ElementContent";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import { getTestId } from "../../tests/test-ids-utils";
 import styles from "./TipseenBasicContent.module.scss";
 
-const BASE_CSS_CLASS = "monday-style-tipseen-basic-content";
-
 interface TipseenBasicContentProps extends VibeComponentProps {
   title?: string;
   titleClassName?: string;
   children?: ElementContent | ElementContent[];
 }
 
-const TipseenBasicContent: FC<TipseenBasicContentProps> = ({ title, children = null, titleClassName, className }) => {
+const TipseenBasicContent: FC<TipseenBasicContentProps> = ({
+  title,
+  children = null,
+  titleClassName,
+  className,
+  id,
+  "data-testid": dataTestId
+}) => {
   return (
-    <div className={cx(styles.tipseenBasicContent, BASE_CSS_CLASS, className)}>
+    <div
+      className={cx(styles.tipseenBasicContent, className)}
+      id={id}
+      data-testid={dataTestId || getTestId(ComponentDefaultTestId.TIPSEEN_CONTENT, id)}
+    >
       <TipseenTitle text={title} className={titleClassName} />
       {children}
     </div>
   );
 };
 
+Object.assign(TipseenBasicContent, {
+  defaultTestId: ComponentDefaultTestId.TIPSEEN_CONTENT
+});
```

Comment:
same here, don't pass default test ids. if you do need it in the subcomponents - propagate it from the parent